static_src/stores/org_store.js

```diff
@@ -17,6 +17,7 @@ export class OrgStore extends BaseStore {
     this._currentOrgGuid = null;
     this._fetchOrg = false;
     this._fetchAll = false;
+    this._cfName = 'org_users';
```

Comment:
@jcscottiii It has to be hardcoded somewhere. This is just moving it to a place where its only hardcoded once.